src/wallet/wallet.h

```diff
@@ -786,6 +786,21 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
     bool FundTransaction(CMutableTransaction& tx, CAmount& nFeeRet, bool overrideEstimatedFeeRate, const CFeeRate& specificFeeRate, int& nChangePosInOut, std::string& strFailReason, bool includeWatching, bool lockUnspents, const std::set<int>& setSubtractFeeFromOutputs, bool keepReserveKey = true, const CTxDestination& destChange = CNoDestination());
 
     /**
+     * Bumps the fee of an opt-in-RBF transaction <txid>,
+     * replacing it with a new transaction <wtxNew>
+     */
+    enum BumpFeeResult
+    {
+        BumpFeeResult_OK,
+        BumpFeeResult_INVALID_ADDRESS_OR_KEY,
+        BumpFeeResult_INVALID_REQUEST,
+        BumpFeeResult_INVALID_PARAMETER,
+        BumpFeeResult_WALLET_ERROR,
+        BumpFeeResult_MISC_ERROR,
+    };
```

Comment:
I though about using exceptions but came to the conclusion that a simple result state enum will (probably) allow more flexibility for further optimisations. But no strong opinion.